egs2/owsm_ctc_v3.1/s2t1/run.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set=train_v3
+valid_set=dev_v3
+test_sets=dev_v3
+
```

Comment:
There is no "data.sh" file in your folder, so you would probably skip the first stage. Maybe mention it by adding a comment here; also add a pointer to owsm 3.1 data prep script.